sdk/search/perf-tests/search-documents/test/getDocumentsCount.spec.ts

```diff
@@ -0,0 +1,251 @@
+import {
+  PerfStressOptionDictionary,
+  PerfStressTest,
+  getEnvVar
+} from "@azure/test-utils-perfstress";
+import {
+  SearchClient,
+  AzureKeyCredential,
+  SearchIndexClient,
+  SearchIndex,
+  KnownAnalyzerNames
+} from "@azure/search-documents";
+import { Hotel } from "./hotel";
+
+export class GetDocumentsCountTest extends PerfStressTest {
+  public options: PerfStressOptionDictionary = {
```

Comment:
If this had been a list test rather than "getting the count" test, you'd have a count parameter here.

Reply:
This test has been removed.